app/views/layouts/application.html.erb

```diff
@@ -16,9 +16,7 @@
     <% unless current_page?(root_path) || current_page?(new_user_registration_path) || current_page?(new_user_session_path) %>
       <%= render 'shared/navbar' %>
       <%= render 'shared/flashes' %>
-      <div class="container">
```

Comment:
If you take the controller from there, you have to add it on every page ... that's not DRY, i think ... ?